.jenkins/agent/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM quay.io/openshift/origin-jenkins-agent-base:latest
-
-COPY install-sdk.sh /install-sdk.sh
-RUN /install-sdk.sh
-RUN dnf install -y ansible && \
+RUN curl -LO "https://github.com/operator-framework/operator-sdk/releases/download/v0.19.4/operator-sdk-v0.19.4-x86_64-linux-gnu" && \
```

Comment:
Note that we're losing a bit of maintenance convenience/safety vs the existing SDK_FULL_NAME envvar usage. Not worth blocking on, but in the future try to remember to preserve the little bit of maintenance sugar where possible, It adds up.